src/qt/rpcconsole.cpp

```diff
@@ -1114,7 +1114,6 @@ void RPCConsole::updateDetailWidget()
         peerAddrDetails += "<br />" + tr("via %1").arg(QString::fromStdString(stats->nodeStats.addrLocal));
     ui->peerHeading->setText(peerAddrDetails);
     ui->peerServices->setText(GUIUtil::formatServicesStr(stats->nodeStats.nServices));
-    ui->peerRelayTxes->setText(stats->nodeStats.fRelayTxes ? ts.yes : ts.no);
```

Comment:
In reality, I think `fNodeStateStatsAvailable` will almost always be true, so this isn't a big issue (if it was, then the other stats in `nodeStateStats` not being available would also be an issue).

Reply:
Oh, this is a bit surprising.  FWIW I found that I needed to add null-checking for the new `m_addr_*` statestats in #22501 to avoid parsing errors and may propose a fix in getpeerinfo.